src/v2/ion/i18nTransformer.js

```diff
@@ -310,4 +310,11 @@ const uiSchemaLabelTransformer = (transformedResp) => {
   return transformedResp;
 };
 
-export { uiSchemaLabelTransformer as default, getMessage, getMessageKey, getI18NParams };
+const isCustomized = (property, settings) => {
```

Comment:
should we rename it, `isCustomizedI18nKey` to be more specific ?

Reply:
rename `property` to `i18nKey` ?